pkg/sql/distsqlrun/indexbackfiller.go

```diff
@@ -135,13 +136,36 @@ func (ib *indexBackfiller) runChunk(
 		sortTime := timeutil.Now().Sub(start)
 
 		start = timeutil.Now()
-		adder, err := ib.flowCtx.BulkAdder(ctx, ib.flowCtx.ClientDB, 32<<20, readAsOf)
+		const maxFlushBytes = 32 << 20
+		flushBytes := int64(len(entries)) * 1000
+		if flushBytes > maxFlushBytes {
```

Comment:
why would we want to flush ssts smaller than max?

Reply:
I did see memory allocation problems which I incorrectly attributed to it. It was the memory allocation in BuildIndexEntriesChunk